packages/0install/0install.2.12/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "talex5@gmail.com"
+authors: "zero-install-devel@lists.sourceforge.net"
+homepage: "http://0install.net/"
+bug-reports: "https://github.com/0install/0install/issues"
+dev-repo: "https://github.com/0install/0install.git"
+build: [
+  [make]
+]
+depends: [
+  "yojson"
+  "xmlm"
+  "ounit"
+  "react"
+  "lwt"
+  "extlib"
+  "ocurl"
+  "sha"
+  "camlp4" {build}
+  "ocamlbuild" {build}
+]
+depopts: [ "obus" "lablgtk" ]
+depexts: [
+  [["ubuntu"] ["unzip"]]
+  [["debian"] ["unzip"]]
+  [["osx" "homebrew"] ["gnupg"]]
```

Comment:
These aren't the same depext and it would be nice if they moved into `conf-*` packages.